### PR TITLE
Update frame.py

### PR DESCRIPTION
**Pull request by trevorkask**
Added an example to the reorder_levels function

- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them
- [ ] whatsnew entry

### CODE REVIEW

**Comment by jbrockmendel** [comment]
the linter has a few complaints

File: pandas/core/frame.py [hunk 1]
```diff
@@ -6788,7 +6788,37 @@ def reorder_levels(self, order: Sequence[Axis], axis: Axis = 0) -> DataFrame:
...
-        """
+        
+        Examples
+        --------
+        >>> data={'class':['Mammals','Mammals','Reptiles','Reptiles','Amphibians','Amphibians'],
+         'diet':['Omnivore','Carnivore','Carnivore','Herbivore','Omnivore','Herbivore'],
+         'species':['Humans','Dogs','Snakes','Iguanas','Frogs','Aquatic Salamanders']
+         }
+
+        >>> df=pd.DataFrame(data,columns=['class','diet','species'])
```
Comment (jreback) [inline pandas/core/frame.py:6799]: space around the equals

File: pandas/core/frame.py [hunk 2]
```diff
@@ -6788,7 +6788,37 @@ def reorder_levels(self, order: Sequence[Axis], axis: Axis = 0) -> DataFrame:
         Returns
         -------
         DataFrame
-        """
+        
+        Examples
+        --------
+        >>> data={'class':['Mammals','Mammals','Reptiles','Reptiles','Amphibians','Amphibians'],
```
Comment (jreback) [inline pandas/core/frame.py:6794]: i don't think this is pep8 formatted, can you add spaces where needed

File: pandas/core/frame.py [hunk 3]
```diff
@@ -6788,7 +6788,37 @@ def reorder_levels(self, order: Sequence[Axis], axis: Axis = 0) -> DataFrame:
...
+         'diet':['Omnivore','Carnivore','Carnivore','Herbivore','Omnivore','Herbivore'],
+         'species':['Humans','Dogs','Snakes','Iguanas','Frogs','Aquatic Salamanders']
+         }
+
+        >>> df=pd.DataFrame(data,columns=['class','diet','species'])
+
+        >>> df.set_index(['class','diet'],drop=True,inplace=True)
+
+        >>> df.reorder_levels(["diet", "class"])
```
Comment (jreback) [inline pandas/core/frame.py:6803]: is this how it actually renders?

**Review by jreback — CHANGES_REQUESTED** [review]
ping when green

**Comment by jreback** [comment]
closing in favor of #42935

**Pull request closed without merging**